### ISSUE TITLE
Death knight stranded at the last quest of the starting zone

### ISSUE THREAD

**Issue by bondrasi** [dalaranwow/dalaran-wow#5334]
![wowscrnshot_030618_161355](https://user-images.githubusercontent.com/37126575/37066277-8d0f7336-2159-11e8-8b9a-b7aa5064872f.jpg)
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
I took the quest at the end of the starting zone for death knights given by the Litch King (fighting at light hope's chapel) but the humans never spawned. Eventually the scourge spawned and I never received credit for the quest. I dropped the quest hoping to reset everything, I also logged off for 7 hours. I can't retake the quest and my toon is stranded at the end of the DK zone.

**Expected behaviour**: Tell us what should happen instead
The quest should have began a battle giving me the chance to finish the zone and enter the world.

**Steps to reproduce the problem**:

1. Take the final quest in the DK starting area to assault light hope's chapel
2. Wait for battle to begin next to the 3 Death Knight's on horses
3. Drop the quest the scourge army despawns and run back to the Litch King to retake the quest

**Include proofs for this behaviour**
I have attached a screenshot of my toon next to the Litch King without the final quest avalible and without me having the quest. I hope this is considered valid proof of my situation.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
I don't know how to discover this information. My toon's name is "Vorkthedork" on Algalon

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
![wowscrnshot_030618_161355](https://user-images.githubusercontent.com/37126575/37066992-aa44b9ea-215c-11e8-826d-03d96dd781b2.jpg)

**Comment by MalkaviaDW** [comment]
Hello there, 

The quest chaining should work fine, I believe it will be solved if you take the quest: "The Light of Dawn" Scourge Commander Thalanor in Browman Mill, and then you have to talk to Darion Mograine in order to start the battle.

Let us know if you still have issues with it.

Cheers!
